### PR TITLE
Create AppStoreDescription

### PR DESCRIPTION
**Pull request by Echolon**
I worked over the text a little bit. Will expand the XEPs by time.

### CODE REVIEW

File: AppStoreDescrition [hunk 1]
```diff
@@ -0,0 +1,25 @@
+There has never been a better time to get into XMPP, a free public chat network that no one controls or owns. 
+Monal is a fast and user friendly way to use the decentral XMPP protocol (known as Jabber). Just download the app, login or register and you are ready
```
Comment (eeeeeta) [inline AppStoreDescrition:2]: nit: `decentral` -> `decentralized`

File: AppStoreDescrition [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+- Open Source and free software (BSD 2-Clause License)
+- No Ads! Strong focus on privacy. Monal does not track you and does not have data metric software 
+included and does not read any personal information. 
+- With the XMPP protocol a direct connection to your server ensures you can keep control of your data.
+- Encrypted chat with OMEMO
+- Monal will work with corporate XMPP servers that require a VPN
+- MUC multi-user chat integration
+
+The software is availbale on iOS and MacOS.
```
Comment (eeeeeta) [inline AppStoreDescrition:15]: nit: `available`

File: AppStoreDescrition [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Monal implements XEP modules and extensions to ensure an enjoying mobile communication experience:
```
Comment (eeeeeta) [inline AppStoreDescrition:17]: nit: `enjoying` -> `enjoyable`

**Review by eeeeeta — CHANGES_REQUESTED** [review]

File: AppStoreDescrition (hunk 3 above)
Comment (Echolon) [reply AppStoreDescrition]: Thank you for reviewing!

**Comment by Echolon** [comment]
Hi Anu,

may you merge it and put it to the App Store description?

**Review by anurodhp — APPROVED** [review]

**Comment by Echolon** [comment]
@anurodhp please do not forget to also update in the App store (or automate this pull if possible)

**Pull request merged**